### ISSUE TITLE
validation loss looks like overfitting

### ISSUE THREAD

**Issue by Maryamzk** [bnsreenu/python_for_microscopists#55]
Hi, 
I have applied this model to other data sets. 
I want to forecast the energy consumption using a history of energy consumption and 23 weather features over 4 years. 

The validation loss is increasing, which I assume it is a sign of overfitting. and the training loss does not go under 0.2. 
I have tried decreasing the learning rate, adding decay rate, and reducing the lstm layers, but I still have overfitting. 

How can I modify my model to prevent overfitting (the increasing trend in the validation loss?
![image](https://user-images.githubusercontent.com/12936292/208121910-c6f5e3e1-fa15-4a3d-b1c5-27275b7908a2.png)

Thanks for any advice.
